src/api/app/mixins/webui/manage_relationships.rb

```diff
@@ -29,6 +23,10 @@ def save_person_or_group(what)
     end
   end
 
+  def users_path
+    url_for(action: :users, project: @package ? @project : @project.name, package: @package || nil)
```

Comment:
what do you think ` || nil` does to the value? :)

Reply:
why easy if there is also a comlicated solution :D thanks, long day ;)